client/src/Components/Pages/Static/CookiesPolicy.js

```diff
@@ -204,7 +204,7 @@ export default class CookiesPolicy extends Component {
                 </Li>
                 <Li>
                   Alternatively you can unsubscribe from our mailing list by
-                  contacting us at hello@earwig.work
+                  contacting us at admin@earwigwork.com
```

Comment:
the issues and the pdf says this should be `hello@earwigwork.com` not `admin@earwigwork.com`
this goes for the other places.

Reply:
don't mind me, I didn't follow the other comments